Installer.sh

```diff
@@ -44,6 +45,7 @@ while [ $# -gt 0 ] ; do
             echo ""
             echo "options:"
             echo "-b, --branch                                [optional] Branch to build Jazz framework from. Defaults to `master`"
+            echo "-codeq, --codequality                       [optional] Code Quality. Defaults to `true`"
```

Comment:
the short of codequality should be shorter. make it `-c` for now

Reply:
The message should be [optional] Install code quality module (sonarqube). Defaults to `false`

Reply:
Make this flag false by default